lnwallet/interface_test.go

```diff
@@ -321,8 +321,10 @@ func createTestWallet(tempTestDir string, miningNode *rpctest.Harness,
 		return nil, err
 	}
 
-	wallet, err := lnwallet.NewLightningWallet(cdb, notifier, wc, signer,
-		bio, netParams)
+	cfg := &lnwallet.Config{}
```

Comment:
Similarly, the changes to this file can be reverted as the wallet doesn't need to know of this new config paramter.